linkml/generators/graphqlgen.py

```diff
@@ -50,13 +53,38 @@ def visit_class_slot(self, cls: ClassDefinition, aliased_slot_name: str, slot: S
             slotrange = slotrange + "!"
         return f"\n    {lcamelcase(aliased_slot_name)}: {slotrange}"
 
+    def visit_enum(self, enum: EnumDefinition):
+        if not enum.permissible_values:
+            if self.fail_on_missing_permissible_values:
+                raise NotImplementedError(
+                    "Enumerations using `reachable_from` instead of `permissible_values` to "
+                    + "specify permissible values is not supported yet."
+                )
+            else:
+                print(
+                    "Warning!! Enumerations using `reachable_from` instead of `permissible_values` "
+                    + f"are not supported yet. Enumeration '{enum.name}' will be simply ignored.",
+                    file=sys.stderr,
+                )
+                return ""
+        values = "\n    ".join([v.upper() for v in enum.permissible_values])
+        out = f"enum {camelcase(enum.name).replace(' ','')}\n  {{\n    {values}\n  }}\n\n"
```

Comment:
I still think it would be a nice thing to make a specific method or regex pattern thats like "format name" and "format value" because the inline formatting can get a bit ad-hoc - eg. This camelcase call is different than the one in `visit_class_slot` and I could see that inconsistency compounding. But I wont insist on it